client/cli/utils.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/pflag"
+
+	"cosmossdk.io/x/upgrade/types"
+)
+
+// parsePlan was copied from the SDK because it was not exported
+// https://github.com/cosmos/cosmos-sdk/blob/x/upgrade/v0.1.4/x/upgrade/client/cli/parse.go
```

Comment:
Small nit, feel free to ignore!

```suggestion
// parsePlan is copied from the Cosmos SDK because it is not exported.
//
// https://github.com/cosmos/cosmos-sdk/blob/x/upgrade/v0.1.4/x/upgrade/client/cli/parse.go#L9-L21
```